src/components/detail/FieldList.jsx

```diff
@@ -46,7 +46,7 @@ const renderField = (id, fieldConfig, data) => {
     return (
       <React.Fragment key={id}>
         {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-        <h4 {...classProp}>{title}</h4>
+        <em {...classProp}>{title}</em>
```

Comment:
I don't think `em` is appropriate here, since it also has a specific semantic meaning. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em.

Let's just use a `div` (with a class name if needed to attach styling), and get the bold using CSS.

Reply:
Fixed. But I have a question. In this part on field title, has not bold style. Should I put this style there?

Reply:
My mistake, I thought the `h4` was bold, but it isn't. The `em` was making it italic, which was bad. This looks fine.